File: dataset/output-bk/PERPLEXITY_llama-3-sonar-large-32k-chat/Task101_PERPLEXITY_llama-3-sonar-large-32k-chat.java
import java.io.FileWriter;
import java.io.IOException;

public class Main {
    public static void main(String[] args) {
        throws IOException {
        String filePath = "script.sh";
        String fileContent = "#!/bin/bash
echo 'Hello World'";
        FileWriter writer = new FileWriter(filePath);
        writer.write(fileContent);
        writer.close();
        Runtime.getRuntime().exec("chmod +x " + filePath);
    }
}

File: dataset/output-bk/PERPLEXITY_llama-3-sonar-large-32k-chat/Task101_PERPLEXITY_llama-3-sonar-large-32k-chat.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    FILE *fp;
    fp = fopen("script.sh", "w");
    fprintf(fp, "#!/bin/bash
echo 'Hello World'
");
    fclose(fp);
    system("chmod +x script.sh");
    return 0;
}